component/background/RoomService.tsx

```diff
@@ -50,19 +54,17 @@ class RoomService {
     });
 
     // set up configRef
-    this.userListRef = this.rootRef.child("user");
-    this.myConfRef = this.userListRef.child(this.id);
-    this.myConfRef.set(true);
-
+    this.rootRef.child(USER).child(this.id).set(true);
```

Comment:
Perhaps break this apart.

```javascript
const rootUser = this.rootRef.child(USER); // apply appropriate name 
const selfRootUser = rootUser.child(this.id);
selfRootUser.set(true);
```

This could help other developers to make sense of what this chaining call is doing.